lib/quickbooks-ruby.rb

```diff
@@ -219,4 +228,12 @@ def initialize(msg)
     end
   end
 
+  class Configuration
+    def self.set_tokens_and_realm_id token, secret, realm_id
+      Thread.current[:token]    = token
```

Comment:
My guess is that `Thread.current` is a safe place to store what is essentially treated as a global variable, without using standard Ruby global variables like `$secret` which pollute the whole namespace.